flytekit/models/sagemaker/parameter_ranges.py

```diff
@@ -199,3 +207,73 @@ def from_flyte_idl(cls, pb2_object: _idl_parameter_ranges.ParameterRanges):
                 converted[k] = CategoricalParameterRange.from_flyte_idl(v.categorical_parameter_range)
 
         return cls(parameter_range_map=converted,)
+
+
+class ParameterRangeOneOf(_common.FlyteIdlEntity):
```

Comment:
Could you add a unit test for this? I added a set of unittests for SageMaker models in my latest PR

Reply:
Done 👍